components/channel_select/channel_select.test.tsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import ChannelSelect from 'components/channel_select/channel_select';
+import Constants from 'utils/constants';
+
+describe('components/ChannelSelect', () => {
+    const defaultProps = {
+        channels: [
```

Comment:
Is there an easy way to generate mock data for the mattermost-redux type `Channel[]` for the unit test?

Reply:
Yes! We have a module in `utils/test_helper` that contains a bunch of mocked out objects, including `Channel`. You can pass in whatever parameters you need to change and it'll generate the full mocked object for you.